gcloud.go

```diff
@@ -91,10 +99,13 @@ func createVMs(names []string, lifetime time.Duration) error {
 		"--scopes", "https://www.googleapis.com/auth/devstorage.read_only,https://www.googleapis.com/auth/logging.write,https://www.googleapis.com/auth/monitoring.write,https://www.googleapis.com/auth/servicecontrol,https://www.googleapis.com/auth/service.management.readonly,https://www.googleapis.com/auth/trace.append",
 		"--image", "ubuntu-1604-xenial-v20171002",
 		"--image-project", "ubuntu-os-cloud",
-		"--boot-disk-size", "50",
+		"--boot-disk-size", "10",
 		"--boot-disk-type", "pd-ssd",
+		"--local-ssd", "interface=SCSI",
```

Comment:
Did you want to make this optional?

Reply:
Yup. Done, as well as `--machine-type`. Added a `VMOpts` to pass everything around.